gammapy/modeling/fit.py

```diff
@@ -25,10 +25,6 @@ class Registry:
 
     Gives users the power to extend from their scripts.
     Used by `Fit` below.
-
-    Not sure if we should call it "backend" or "method" or something else.
-    Probably we will code up some methods, e.g. for profile analysis ourselves,
-    using scipy or even just Python / Numpy?
```

Comment:
Why did you remove this note ?
I would suggest to keep it and have an other PR to decide what to do this. 
We may have to change the name as we have already have a registry class in `gammapy/utils/registry.py`.
Also this `Registry` is not public API so the sentence above that say it allows user to extend supported backends is not really true.